web/src/components/speaker/Links.vue

```diff
@@ -1,29 +1,56 @@
 <template>
   <div class="mt-3">
-    <social-button :link="twitter">
-      <i class="fab fa-twitter" />
-    </social-button>
-    <social-button :link="linkedIn">
-      <i class="fab fa-linkedin-in" />
-    </social-button>
-    <social-button :link="facebook">
-      <i class="fab fa-facebook-f" />
-    </social-button>
-    <social-button :link="website">
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
+      <template v-slot:activator="{ on }">
+        <social-button :link="twitter">
+          <i class="fab fa-twitter" v-on="on" />
+        </social-button>
+      </template>
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
-      <v-icon>web</v-icon>
-    </social-button>
-    <social-button :link="priorPresentation">
-      <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
-      <v-icon>ondemand_video</v-icon>
-    </social-button>
+      <span>Twitter</span>
+    </v-tooltip>
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
+      <template v-slot:activator="{ on }">
+        <social-button :link="linkedIn">
+          <i class="fab fa-linkedin-in" v-on="on" />
+        </social-button>
+      </template>
+      <span>LinkedIn</span>
+    </v-tooltip>
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
+      <template v-slot:activator="{ on }">
+        <social-button :link="facebook">
+          <i class="fab fa-facebook-f" v-on="on" />
+        </social-button>
+      </template>
+      <span>Facebook</span>
+    </v-tooltip>
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
+      <template v-slot:activator="{ on }">
+        <social-button :link="website">
+          <v-icon v-on="on">web</v-icon>
+        </social-button>
+      </template>
+      <span>Website</span>
+    </v-tooltip>
+    <v-tooltip top :disabled="$vuetify.breakpoint.width < 600">
+      <template v-slot:activator="{ on }">
+        <social-button :link="priorPresentation">
+          <v-icon v-on="on">ondemand_video</v-icon>
+        </social-button>
+      </template>
+      <span>Presentation</span>
+    </v-tooltip>
   </div>
 </template>
 
 <script>
 import SocialButton from '@/components/buttons/SocialButton.vue';
 
 export default {
+  mounted() {
```

Comment:
Great for testing, but let's remove before merging to master 😄

Reply:
This was removed in the most recent commit. Not sure why it still shows here. 😅 Sorry!

Reply:
GH will show it with the "Outdated" note, all good 👍🏻